sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_ModelCompose.md

```diff
@@ -0,0 +1,86 @@
+# Model compose
+
+Model compose allows multiple models to be composed and called with a single model ID. This is useful when you have created different models and want to aggregate a group of them into a single model that you (or a user) could use to analyze a document.
+When doing so, you can let the service decide which model more accurately represents the document, instead of manually trying each  model against the document and selecting the most appropiate one.
```

Comment:
nit: double space in between "each" and "model" in "...manually trying each  model against..."

Reply:
wow. great eyes!